lib/audit/catalog_to_moab.rb

```diff
@@ -133,4 +134,11 @@ def update_status(new_status)
     end
   end
 
+  def online_moab_found?(druid, storage_dir)
+    @moab ||= begin
+      object_dir = "#{storage_dir}/#{DruidTools::Druid.new(druid).tree.join('/')}"
+      Moab::StorageObject.new(druid, object_dir)
+    end
+  end
```

Comment:
just because of the name of the method, what about having this return a proper boolean value instead of a truthy/falsey value?  (since it seems to return the moab or `nil` here, right?)

Reply:
can i change it to `online_moab`?